src/docfx/build/toc/parser/TableOfContentsInputItem.cs

```diff
@@ -57,8 +58,9 @@ public static TableOfContentsItem ToTableOfContentsModel(TableOfContentsInputIte
                 Expanded = inputModel.Expanded,
                 ExtensionData = inputModel.ExtensionData,
                 Children = children?.ToList(),
-                Monikers = monikers,
             };
+            tocItem.Monikers.AddRange(monikers);
```

Comment:
>tocItem.Monikers.AddRange(monikers); [](start = 12, length = 36)

v3 seems to prefer assignment instead of modify a property in place. Seems we an keep `set` for it.

Reply:
ok